### ISSUE TITLE
Add support for Python 3

### ISSUE THREAD

**Issue by tdruez** [nexB/license-expression#1]
```
File "lib/python3.5/site-packages/license_expression.py", line 324, in build
    if isinstance(expression, basestring) and expression.strip():
NameError: name 'basestring' is not defined
```

Could be solved with the following:
```
try:
    unicode = unicode
except NameError:
    str = str
    unicode = str
    bytes = bytes
    basestring = (str, bytes)
else:
    str = str
    unicode = unicode
    bytes = str
    basestring = basestring
```

**Comment by pombredanne** [comment]
Fixed with #3 . Next is to add a test matrix for Travis to test on Python3 too

**Comment by pombredanne** [comment]
Fixed in master
